src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2153,7 +2164,8 @@ internal void RecordImport(ExternAliasDirectiveSyntax syntax)
 
         private void RecordImportInternal(CSharpSyntaxNode syntax)
         {
-            LazyInitializer.EnsureInitialized(ref _lazyImportInfos).
+            // Note: the suppression will be unnecessary once LazyInitializer is properly annotated
```

Comment:
Anything tracking this?

Reply:
This is a BCL API which is not in mscorlib (`System.Threading`). I'm expecting they will just get to it for .NET 5.